cmd/kube-scheduler/app/options/options.go

```diff
@@ -184,6 +185,9 @@ func (o *Options) ApplyTo(c *schedulerappconfig.Config) error {
 func (o *Options) Validate() []error {
 	var errs []error
 
+	if err := validation.ValidateKubeSchedulerConfiguration(&o.ComponentConfig).ToAggregate(); err != nil {
```

Comment:
If I upgrade a cluster from 1.11, can a previously working scheduler start to fail because of validation?  Ie can this validation cause failures on upgrade?

Reply:
Sorry, im not familiar with the kubeadm upgrade process, but I would assume yes. 

@bsalamat should know more

Reply:
I do not expect failures. The validations we are doing here are basic checks to ensure that the values are in the expected range. These validations would fail only when the provided config do not comply with the documented values in the API.